src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -39,6 +39,17 @@ public CacheOptions(bool useSharedCache)
             UseSharedCache = useSharedCache;
         }
 
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="useSharedCache">Set to true to share the cache between all ClientApplication objects. The cache becomes static. <see cref="UseSharedCache"/> for a detailed description. </param>
+        /// <param name="sizeLimit">Token cache size limit in bytes. <see cref="SizeLimit"/> for a detailed description.</param>
+        public CacheOptions(bool useSharedCache, long sizeLimit)
+        {
+            UseSharedCache = useSharedCache;
+            SizeLimit = sizeLimit;
+        }
```

Comment:
Suggestion: Declare ATSizeInBytes here as internal static and validate that the value passed is greater than min required.